CIScripts/Build/Zuul.ps1

```diff
@@ -15,7 +15,6 @@ function Clone-ZuulRepos {
         "--zuul-url=$ZuulUrl",
         "--zuul-branch=$ZuulBranch",
         "--map=./CIScripts/clonemap.yml",
-        "--verbose",
         $GerritUrl
```

Comment:
We don't understand why this was removed, could you clarify?

Reply:
To get rid of ZUUL's `Debug` messages (`Info` messages are still present)